packages/core/lib/std.js

```diff
@@ -42,7 +42,7 @@ const defRule = (evalRuleSet) => (scope) => (_opts) => (name, body) => {
 };
 
 /**
- * The root scope that contains all the basic/standard rules and variables
+ * @scope { The root scope that contains all the basic/standard functions and variables }
```

Comment:
are we using `functions` or `rules`? 😛 

You're the one who told me we should be consistent and use `rule` which is why I changed `func` to `rule` everywhere